inc/Engine/Preload/Admin/Settings.php

```diff
@@ -74,6 +74,38 @@ private function can_display_notice(): bool {
 		return $this->is_enabled();
 	}
 
+	/**
+	 * Display missing as table notice if they are not present.
+	 *
+	 * @return void
+	 */
+	public function maybe_display_as_missed_tables_notice() {
+
+		if ( function_exists( 'get_current_screen' ) && 'tools_page_action-scheduler' === get_current_screen()->id ) {
+			return;
+		}
+
+		if ( $this->is_valid_as_tables() ) {
+			return;
+		}
+
+		$as_tools_link = menu_page_url( 'action-scheduler', false );
+		$message       = sprintf(
+		// translators: %1$s = plugin name, %2$s = opening anchor tag, %3$s = closing anchor tag.
+			__( '%1$s: We detected missing database table related to Action Scheduler. Please visit the following %2$sURL%3$s to recreate it, as it is needed for WP Rocket to work correctly.', 'rocket' ),
+			'<strong>WP Rocket</strong>',
+			'<a href="' . $as_tools_link . '">',
```

Comment:
```suggestion
			'<a href="' . menu_page_url( 'action-scheduler', false ) . '">',
```

Can use the function directly there, as the variable is not used more than once